src/app/file-manager/services/gcs-api.service.ts

```diff
@@ -59,44 +54,39 @@ export class GcsApiService extends GcsService {
   }
 
   public cancelAll() {
-    const itemsToDownload = this.getDownloadingFiles();
-    const itemsToUpload = this.getUploadingFiles();
-    const itemsToExport = this.getExportPendingFiles();
-    if (itemsToDownload.length > 0) {
-      this.electronService.ipcRenderer.send('download-cancel');
-      this.cancelItemsStatus(itemsToDownload);
-    }
-    if (itemsToUpload.length > 0) {
-      this.electronService.ipcRenderer.send('upload-cancel');
-      this.cancelItemsStatus(itemsToUpload);
-    }
-    if (itemsToExport.length > 0) {
-      this.electronService.ipcRenderer.send('export-gcp-cancel');
-      this.cancelItemsStatus(itemsToExport);
+    if (new FilesDatabase(this.store).data.filter(item => item.status === ItemStatus.PENDING
```

Comment:
I don't think this logic is correct, or it's at least redundant, as it mentions `PENDING` twice.

I know that `&&` takes precedence over `||` but I find that it's clearer to use `(` `)` around the `&&` expression so there's no confusion, assuming that you need to keep the `&&` part of this expression.

Reply:
This is now fixed